.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+# Copyright The Helm Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
```

```suggestion
    directory: "/"
    groups:
      github-actions:
        patterns:
          - "*"
    schedule:
      interval: "daily"
```

This could be an alternative pattern with Dependabot grouping all GitHub Actions changes into a single PR?

---

sounds good